tools/mock-service-host/script/renew-ssh.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Let me know your thoughts, but I think as a general convention we should aim to have all scripts written once in powershell targeting powershell core. There's a little bit of an investment in learning powershell, but it is guaranteed cross-platform on every developer machine no matter the OS and pipeline agent.

Reply:
Thanks for point out, @benbp.
I added this for Linux/Mac. Since the powershell convention, I will add ps1 version and use it in build pipeline. 

And I think we don't need to delete this sh file in case there are existing mock-host users who are depending on this.